pkg/util/constant.go

```diff
@@ -33,10 +33,12 @@ const (
 	JivaCasType = "jiva"
 	// LVMCasType cas type name
 	LVMCasType = "localpv-lvm"
-	// LocalHostpathCasType cas type name
-	LocalHostpathCasType = "localpv-hostpath"
+	// LocalPvHostpathCasType cas type name
+	LocalPvHostpathCasType = "localpv-hostpath"
```

Comment:
```suggestion
	LocalPvHostpathCasType = "local-hostpath"
```

Update this constant to local-hostpath, and change the variables elsewhere accordingly.